src/config.js

```diff
@@ -4,12 +4,14 @@ var path = require('path');
 
 var config = module.exports;
 
+const CONFIG_VAR_REGEX = /\$([{A-Z])\w+}/g;
+
 /**
  * Load config file with defaults
  * @param filename
  */
-config.load = function(filename = '_config.yml') {
-    const conf = new Config(filename);
+config.load = function(filename = '_config.yml', version='') {
```

Comment:
I really like this for config overrides... later we can replace `versions` with `argv so that it's more flexible